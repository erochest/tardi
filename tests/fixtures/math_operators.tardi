40 2 +
10 3 +

40 2 -
10 3 -