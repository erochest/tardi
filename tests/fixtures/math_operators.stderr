42
13
38
7
80
30
